File: 1911-minimum-elements-to-add-to-form-a-given-sum/minimum-elements-to-add-to-form-a-given-sum.java
class Solution {
    public int minElements(int[] nums, int limit, int goal) {
        long sum = 0;
        for (int num : nums) sum += num;
        long diff = Math.abs(sum-goal);
        return diff % limit == 0L ? (int) ( diff / limit ) : (int) ( diff / limit ) + 1 ;
    }
}

File: 1911-minimum-elements-to-add-to-form-a-given-sum/minimum-elements-to-add-to-form-a-given-sum.kt
class Solution {
    fun minElements(nums: IntArray, limit: Int, goal: Int): Int {
        val sum : Long = nums.sumOf { it.toLong() }
        var diff : Long = goal - sum
        if ( diff < 0 ) diff *= -1
        return if ( diff % limit == 0L ) (diff/limit).toInt() else ((diff/limit)+1).toInt()
    }
}